examples/Atlas/runAtlasFootstepPlanning.m

```diff
@@ -18,7 +18,7 @@
                    'left', forwardKin(r, kinsol, r.foot_frame_id.left, [0;0;0], 1));
 
 % Plan footsteps to the goal
-goal_pos = struct('right', [1;0;0;0;0;0], 'left', [1;0.26;0;0;0;0]);
+goal_pos = struct('right', [1;-0.13;0;0;0;0], 'left', [1;0.13;0;0;0;0]);
```

Comment:
This is unrelated to the fix, but has been wrong for a long time (the footstep planner unit test was using a goal pose which wasn't aligned with the current foot poses, which made the resulting plan weirdly diagonal and looked like the planner was doing something wrong).